web/src/components/onboarding/DevOnlySkipOnboardingButton.tsx

```diff
@@ -10,15 +12,21 @@ interface Props {
 
 export const DevOnlySkipOnboardingButton = (props: Props): ReactElement => {
   const { state, setProfileData } = useContext(ProfileDataContext);
+  const { updateQueue } = useUserData();
```

Comment:
[dust/pebble] This is a great change, but you could make this change on a different commit and merge it into main directly because it isn't related to the work in this commit.

Reply:
Done #8430